package.json

```diff
@@ -53,6 +53,7 @@
         "eslint-plugin-prettier": "^4.0.0",
         "hammer": "^0.0.5",
         "hammerjs": "^2.0.8",
+        "jshint": "^2.13.6",
```

Comment:
Why was jshint added?  It is related to hammer and hammerjs being added? What warning?